poandy/controller/trade.py

```diff
@@ -4,7 +4,15 @@
 
 class TradeController(Controller):
     @classmethod
-    def get_trades(cls, account_id, state="OPEN", instrument=None, count=50):
+    def get_trades(
+        cls,
+        account_id,
+        ids=None,
+        state="OPEN",
```

Comment:
Like cal said, I think we should introduce enums. But that is out of scope of this PR for now. Maybe we can do it in a subsequent issue.

Reply:
https://github.com/QAM-ATC/poandy/pull/18#discussion_r578179831

^ yea agreed